src/main/java/duke/utils/Storage.java

```diff
@@ -0,0 +1,137 @@
+package duke.utils;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.Scanner;
+
+import duke.exceptions.TaskCreationException;
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.Todo;
+
+/**
+ * This class implements the saving and reading of bot data into a text file.
+ *
+ * @author delishad21
+ */
+public class Storage {
+    private File f;
```

Comment:
Perhaps a better name can be used?

Reply:
I agree with @alvinnzz. Perhaps the variable name can explain the entity better?